shipmmg/kt.py

```diff
@@ -103,15 +296,17 @@ def simulate(
     """
```

Comment:
It is difficult to understand how to use this OdeSolusion.

Here is one of the improving plans of Examples.
```
sol = simulate_kt(kt_params, time_list, delta_list, r0)
result = sol.sol(time_list)
```